### PR TITLE
adds music to enchiridion

### PR DESCRIPTION
**Pull request by jcpirtle**
@ehgoodenough idk how to make a playable build with the --production.

### CODE REVIEW

**Comment by jcpirtle** [comment]
fixes #51

**Comment by ehgoodenough** [comment]
@jcpirtle: Just go into `./share.sh` and change that one line that says `node build` to `node build --production`. Make sure to put it back when you're done.

File: source/index.js [hunk 1]
```diff
@@ -60,6 +62,41 @@ var loop = new Afloop((delta) => {
...
+        yellowTrack.play()
+    }
+
+    yellowTrack.onended = function() {
+        purpleTrack.play()
+    }
+
+    purpleTrack.onended = function() {
+        redTrack.play()
+    }
```
Comment (ehgoodenough) [inline source/index.js]: I like that you made it loop! Nice!

File: source/index.js [hunk 2]
```diff
@@ -60,6 +62,41 @@ var loop = new Afloop((delta) => {
     render(state)
 })
 
+
+
+
+if(STAGE == "PRODUCTION") {
```
Comment (ehgoodenough) [inline source/index.js:66]: Thanks; this'll save my sanity while developing.

**Comment by jcpirtle** [comment]
https://mocsarcade.github.io/enchiridion/musictest

**Review by ehgoodenough — APPROVED** [review]
LGTM

**Comment by ehgoodenough** [comment]
My only concern is how long it takes to load, now that we're importing the music files.

**Pull request merged**